Dockerfile

```diff
@@ -1,3 +1,15 @@
+FROM node:10 as frontend-builder
+
+COPY package.json package-lock.json webpack.config.js /frontend/
+COPY client/ /frontend/client
```

Comment:
Maybe copy the whole path, just to be on the safe side? It's a temporary container anyway.